db/create_tables.sql

```diff
@@ -39,7 +39,7 @@ CREATE TABLE IF NOT EXISTS `events` (
   `type` VARCHAR(100) NOT NULL,
   `status` VARCHAR(100) DEFAULT NULL,
   `state_machine_instance_id` VARCHAR(255) NOT NULL,
```

Comment:
why have a VARCHAR() without having any referential integrity?
Why not link it with PK of `state_machine` table